apps/src/music/views/TimelineTrackEvents.jsx

```diff
@@ -83,6 +84,9 @@ const TimelineTrackEvents = ({
                         barWidth={barWidth}
                         height={singleElementHeight}
                         top={index * singleElementHeight}
+                        left={0}
```

Comment:
Added this since `left` was coming through as `undefined`.

Reply:
Ok with this change, but just curious, does `left` need to be a required prop in `TimelineElement`?

Reply:
The initial idea was that the element would be positioned using this value.  Did that not work well in tracks mode?

Reply:
Yep, it's just used one level higher in tracks mode ([here](https://github.com/code-dot-org/code-dot-org/blob/c4a00c7503ab5edfcae4b10259450ca9ede73539/apps/src/music/views/TimelineTrackEvents.jsx#L75)) because elements are organized into "stacks" that start at the same time due to the mutator functionality. So the positioning is not needed on the individual element

Reply:
Fair.  I can make it optional again, then.